integration-cli/docker_api_containers_test.go

```diff
@@ -138,11 +138,7 @@ func TestContainerApiStartVolumeBinds(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	bindPath, err := ioutil.TempDir(os.TempDir(), "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-
+	bindPath := path.Join("/tmp", fmt.Sprintf("test.%s", makeRandomString(10)))
```

Comment:
^^ so, this might confuse people, because they *think* it should be replaced by `filepath` (because that's usually the right way); I think a comment should be added to these lines to explain *why* this is used and to prevent people from refactoring them by accident.